androidApp/src/main/java/com/mbta/tid/mbta_app/android/phoenix/PhoenixSocketWrapper.kt

```diff
@@ -20,7 +20,9 @@ value class PhoenixSocketWrapper(private val socket: Socket) : PhoenixSocket {
 
     fun attachLogging() {
         socket.onMessage { message -> Log.i("Socket", message.toString()) }
-        socket.onError { throwable, response -> Log.e("Socket", response.toString(), throwable) }
+        socket.onError { throwable, response ->
+            Log.e("Socket", response?.toString() ?: throwable.toString(), throwable)
+        }
```

Comment:
This is a minor improvement to the way we log errors that aren't sent by the server - for a bit I was getting domain resolution failures, and those were just logging as `null` since `response` was null, and that's not useful.